server/data_cxg/cxg_adaptor.py

```diff
@@ -133,6 +138,15 @@ def isvalid(url):
             return False
         return True
 
+    def diffexp_cache(self, hashval):
+        """Find the hashval in the cache.  For now, the cache is only populated from the precomputed values"""
+        if not self.diffexp_has_cache:
+            return None
+        if self.diffexp_hashkey is None:
+            hashkey = self.open_array("diffexp/hashkey")
+            self.diffexp_hashkey = {h: i for i, h in enumerate(hashkey[:])}
```

Comment:
why pre-load (memory cache) this index, but not the mean & var?  They are not going to take much more space than the dict itself....